### ISSUE TITLE
Composer not actually supported

### ISSUE THREAD

**Issue by SteveEdson** [WadeShuler/PHP-PayPal-IPN#2]
Although this can be loaded through composer, it will not be autoloaded.

The following code will fail with a fatal error:

```php
<?php 

require "vendor/autoload.php";

$listener = new IPNListener();

```

> Fatal error: Class 'IPNListener' not found 

Thanks.

**Comment by WadeShuler** [comment]
I am still booked. Sorry, I just am on a few projects and really don't have much spare time right now.

Do you know how to fix it? If so, put in another merge request and I will merge it. Thanks.

**Comment by WadeShuler** [comment]
Thanks to @antriver, this is solved. He fixed it up for Composer via #5
